jsyntrax/src/main/java/org/atpfivt/jsyntrax/InputArguments.java

```diff
@@ -39,6 +41,16 @@
         optionsMap.put(new Option(null, "scale", true, "Scale image"),
                 (o, s) -> o.scale = Double.parseDouble(s)
         );
+        optionsMap.put(new Option(null, "get-style", false, "Get default style config"),
+                (o, s) -> {o.getDefaultStyle = true;}
+        );
+        optionsMap.put(new Option("v", "version", false, "Release version"),
+                (o, s) -> {
+                    o.version = Main.class
```

Comment:
Usually `--version` key returns program name AND its version

in our case this should be `"JSyntrax " + Main.class` ...